scripts/build_podspecs.sh

```diff
@@ -75,7 +75,7 @@ for target in "${targets[@]}"; do
 
   while read -r raw_dependency; do
     dependencies+=( "${newline}  s.dependency '$raw_dependency', s.version.to_s" )
-  done < <("${here}/list_topsorted_dependencies.sh" -d "${target#Swift}" | grep -v "NIOPriorityQueue" | sed 's/^NIO/SwiftNIO/')
+  done < <("${here}/list_transitive_dependencies.py" "${target#Swift}" | grep -v "NIOPriorityQueue" | sed 's/^NIO/SwiftNIO/')
```

Comment:
should we just drop the `.sh`/`.py` suffix? It doesn't really help and causes issues in situations like this...

Reply:
Is it causing an issue? We're calling a different script entirely, not the same one rewritten in a different language.

Reply:
Generally I also have a slightly preference to not having the prefix for scripts _but_ the extension is used by the soundness script when validating the license header so it does serve a small purpose here.

Reply:
Oh shoot, that's probably also why I named it that in the first place. Not sure if that's a good reason but at least there is a reason :). So yes, consider my point here moot.